packages/core/src/types.ts

```diff
@@ -4,6 +4,17 @@ export type TCCSSCoreProp = {
 export type TNestedInput = { [key: string]: TNestedInput | string | number }
 export type TCSSPropValue = <T>(v: string | string[] | TNestedInput[] | TNestedInput, o?: T) => string
 export type TCSSSimplePropValue = <T>(v?: T | TNestedInput | string) => T | string | undefined
+export type TMediaQueryInput = [EBreakpoint | string, string][]
+export type TMediaQueryFunc = (input: TMediaQueryInput) => string | undefined
+
+export enum EBreakpoint {
```

Comment:
We cannot provide this, this is just the default, user can have any breakpoint set.

Reply:
removing